// tailwindcss/tmp.js

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 151
Content-Type: application/octet-stream
X-File-MD5: 8f0f4f41e97516c397218dddacb2a95e
X-File-Mtime: 1691012381
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/plugin.js

let createPlugin = require('./lib/public/create-plugin')
module.exports = (createPlugin.__esModule ? createPlugin : { default: createPlugin }).default

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 446
Content-Type: application/octet-stream
X-File-MD5: 3915d752dc600c6a21861f4ea7e7f457
X-File-Mtime: 1691012381
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/prettier.config.js

module.exports = {
  // These settings are duplicated in .editorconfig:
  tabWidth: 2, // indent_size = 2
  useTabs: false, // indent_style = space
  endOfLine: 'lf', // end_of_line = lf
  semi: false, // default: true
  singleQuote: true, // default: false
  printWidth: 100, // default: 80
  trailingComma: 'es5',
  bracketSpacing: true,
  overrides: [
    {
      files: '*.js',
      options: {
        parser: 'flow',
      },
    },
  ],
}

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 20
Content-Type: application/octet-stream
X-File-MD5: c8bb64bc43755eefb05fb38a4ea4784c
X-File-Mtime: 1691012380
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/variants.css

@tailwind variants;

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 123
Content-Type: application/octet-stream
X-File-MD5: a0606594b7babaae92fcab7211ed65f6
X-File-Mtime: 1691012381
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/resolveConfig.d.ts

import type { Config } from './types/config'
declare function resolveConfig(config: Config): Config
export = resolveConfig

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 156
Content-Type: application/octet-stream
X-File-MD5: 8db25c0a907db5424a5cf50a6d1bb7d8
X-File-Mtime: 1691012381
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/resolveConfig.js

let resolveConfig = require('./lib/public/resolve-config')
module.exports = (resolveConfig.__esModule ? resolveConfig : { default: resolveConfig }).default

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 21
Content-Type: application/octet-stream
X-File-MD5: 6f59153df1de24ba3140c3b2c008a855
X-File-Mtime: 1691012380
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/utilities.css

@tailwind utilities;

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 29
Content-Type: application/octet-stream
X-File-MD5: c1d403dcf755d22cfea715fba2b2ecfa
X-File-Mtime: 1691012381
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/tmp.dependency-graph.js

1. Push tag
2. Draft release

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 61
Content-Type: application/octet-stream
X-File-MD5: 9acd34157f4fab8d4db3ea978ea58f27
X-File-Mtime: 1691012380
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/tailwind.css

@tailwind base;

@tailwind components;

@tailwind utilities;

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 59
Content-Type: application/octet-stream
X-File-MD5: c040ed42da55bbd3300ce3fb11cfc458
X-File-Mtime: 1691012380
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/tmp.in.css

@tailwind base;
@tailwind components;
@tailwind utilities;

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 1091
Content-Type: application/octet-stream
X-File-MD5: b561e0a423bedc9d9ca9c8c67d40abb9
X-File-Mtime: 1691012380
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/to-regex-range/LICENSE

The MIT License (MIT)

Copyright (c) 2015-present, Jon Schlinkert.

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in
all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
THE SOFTWARE.

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 320
Content-Type: application/octet-stream
X-File-MD5: 02e75f07df00367a43d8e50a7f8ae48c
X-File-Mtime: 1691012380
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/tailwindcss/tmp.css

@media (prefers-color-scheme: dark) {
  @supports (hover: hover) {
    .dark\:\[\@supports\(hover\:hover\)\]\:hover\:\[\&\>\*\]\:\(\[--potato\:baked\]\2c
      bg-\[\#0088cc\]\)
      > *:hover {
      --tw-bg-opacity: 1;
      background-color: rgb(0 136 204 / var(--tw-bg-opacity));
      --potato: baked;
    }
  }
}

--boundary_.oOo._5vG5bheTy4KBcp+OjmfdqXOT6wO7jamZ
Content-Length: 6481
Content-Type: application/octet-stream
X-File-MD5: 4a8c07ceb1974121b3e401d0274e1527
X-File-Mtime: 1691012380
X-File-Path: /project-polref/Encyclopedia/website_polar/node_modules/to-regex-range/index.js

/*!
 * to-regex-range <https://github.com/micromatch/to-regex-range>
 *
 * Copyright (c) 2015-present, Jon Schlinkert.
 * Released under the MIT License.
 */

'use strict';

const isNumber = require('is-number');

const toRegexRange = (min, max, options) => {
  if (isNumber(min) === false) {
    throw new TypeError('toRegexRange: expected the first argument to be a number');
  }

  if (max === void 0 || min === max) {
    return String(min);
  }

  if (isNumber(max) === false) {
    throw new TypeError('toRegexRange: expected the second argument to be a number.');
  }

  let opts = { relaxZeros: true, ...options };
  if (typeof opts.strictZeros === 'boolean') {
    opts.relaxZeros = opts.strictZeros === false;
  }

  let relax = String(opts.relaxZeros);
  let shorthand = String(opts.shorthand);
  let capture = String(opts.capture);
  let wrap = String(opts.wrap);
  let cacheKey = min + ':' + max + '=' + relax + shorthand + capture + wrap;

  if (toRegexRange.cache.hasOwnProperty(cacheKey)) {
    return toRegexRange.cache[cacheKey].result;
  }

  let a = Math.min(min, max);
  let b = Math.max(min, max);

  if (Math.abs(a - b) === 1) {
    let result = min + '|' + max;
    if (opts.capture) {
      return `(${result})`;
    }
    if (opts.wrap === false) {
      return result;
    }
    return `(?:${result})`;
  }

  let isPadded = hasPadding(min) || hasPadding(max);
  let state = { min, max, a, b };
  let positives = [];
  let negatives = [];

  if (isPadded) {
    state.isPadded = isPadded;
    state.maxLen = String(state.max).length;
  }

  if (a < 0) {
    let newMin = b < 0 ? Math.abs(b) : 1;
    negatives = splitToPatterns(newMin, Math.abs(a), state, opts);
    a = state.a = 0;
  }

  if (b >= 0) {
    positives = splitToPatterns(a, b, state, opts);
  }

  state.negatives = negatives;
  state.positives = positives;
  state.result = collatePatterns(negatives, positives, opts);

  if (opts.capture === true) {
    state.result = `(${state.result})`;
  } else if (opts.wrap !== false && (positives.length + negatives.length) > 1) {
    state.result = `(?:${state.result})`;
  }

  toRegexRange.cache[cacheKey] = state;
  return state.result;
};

function collatePatterns(neg, pos, options) {
  let onlyNegative = filterPatterns(neg, pos, '-', false, options) || [];
  let o
